opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -503,9 +500,107 @@ message DoubleHistogramDataPoint {
   // a boolean value which decides what type of intervals to use.
   repeated double explicit_bounds = 7;
 
+  // When bounds have a pattern, a more efficient encoding method may be used.
+  // Only one method should be defined. For performance reason, "oneof" is not used.
+  // When more than one methods are defined, the receiver will use the first one found using this order:
+  //     explicit_bounds, linear_bounds, exponential_bounds, log_linear_bounds
+  LinearBounds linear_bounds = 9;
+  ExponentialBounds exponential_bounds = 10;
+  LogLinearBounds log_linear_bounds = 11;
+
   // (Optional) List of exemplars collected from
   // measurements that were used to form the data point
   repeated DoubleExemplar exemplars = 8;
+
+  // LinearBounds. Bounds can be generated via
+  // for (int i = 0; i < num_of_bounds; i++)
+  //     boundList.add(offset + width * i);
+  message LinearBounds {
+    double offset = 1;
+    double width = 2;
+    uint32 num_of_bounds = 3;
+  }
+
+  // ExponentialBounds. Bounds are on log scale.
+  // The bound sequence is stitched together using negative bounds, zero bound, and positive bounds.
+  // The formula to generate explicit bounds here are for demonstration purpose only.
+  // If the message receiver natively stores exponential histograms, it will not need to generate explicit bounds.
+  // It will only need to record the exponential bound parameters.
+  message ExponentialBounds {
+    double reference = 1;
```

Comment:
May we added that if `reference` is omitted, it's value may be presumed to equal 1?

Reply:
comment accepted.